presto-hive/src/main/java/io/prestosql/plugin/hive/HiveStorageFormat.java

```diff
@@ -38,75 +40,92 @@
-            DataSize.of(128, Unit.MEGABYTE)),
+            DataSize.of(128, Unit.MEGABYTE),
+            ".parquet"),
     AVRO(
             AvroSerDe.class.getName(),
             AvroContainerInputFormat.class.getName(),
             AvroContainerOutputFormat.class.getName(),
-            DataSize.of(64, Unit.MEGABYTE)),
+            DataSize.of(64, Unit.MEGABYTE),
+            ".avro"),
     RCBINARY(
             LazyBinaryColumnarSerDe.class.getName(),
             RCFileInputFormat.class.getName(),
             RCFileOutputFormat.class.getName(),
-            DataSize.of(8, Unit.MEGABYTE)),
+            DataSize.of(8, Unit.MEGABYTE),
+            ".rc"),
     RCTEXT(
             ColumnarSerDe.class.getName(),
             RCFileInputFormat.class.getName(),
             RCFileOutputFormat.class.getName(),
-            DataSize.of(8, Unit.MEGABYTE)),
+            DataSize.of(8, Unit.MEGABYTE),
+            ".rc"),
     SEQUENCEFILE(
             LazySimpleSerDe.class.getName(),
             SequenceFileInputFormat.class.getName(),
             HiveSequenceFileOutputFormat.class.getName(),
-            DataSize.of(8, Unit.MEGABYTE)),
+            DataSize.of(8, Unit.MEGABYTE),
+            ".seq"),
     JSON(
             JsonSerDe.class.getName(),
             TextInputFormat.class.getName(),
             HiveIgnoreKeyTextOutputFormat.class.getName(),
-            DataSize.of(8, Unit.MEGABYTE)),
+            DataSize.of(8, Unit.MEGABYTE),
+            ".json"),
     TEXTFILE(
             LazySimpleSerDe.class.getName(),
             TextInputFormat.class.getName(),
             HiveIgnoreKeyTextOutputFormat.class.getName(),
-            DataSize.of(8, Unit.MEGABYTE)),
+            DataSize.of(8, Unit.MEGABYTE),
+            ".txt"),
     CSV(
             OpenCSVSerde.class.getName(),
             TextInputFormat.class.getName(),
             HiveIgnoreKeyTextOutputFormat.class.getName(),
-            DataSize.of(8, Unit.MEGABYTE));
+            DataSize.of(8, Unit.MEGABYTE),
+            ".csv");
 
     private final String serde;
     private final String inputFormat;
     private final String outputFormat;
     private final DataSize estimatedWriterSystemMemoryUsage;
+    private final String fileExtension;
 
-    HiveStorageFormat(String serde, String inputFormat, String outputFormat, DataSize estimatedWriterSystemMemoryUsage)
+    HiveStorageFormat(String serde, String inputFormat, String outputFormat, DataSize estimatedWriterSystemMemoryUsage, String fileExtension)
     {
         this.serde = requireNonNull(serde, "serde is null");
         this.inputFormat = requireNonNull(inputFormat, "inputFormat is null");
         this.outputFormat = requireNonNull(outputFormat, "outputFormat is null");
         this.estimatedWriterSystemMemoryUsage = requireNonNull(estimatedWriterSystemMemoryUsage, "estimatedWriterSystemMemoryUsage is null");
+        this.fileExtension = requireNonNull(fileExtension, "fileExtension is null");
```

Comment:
Instead of having it as `String` can we have a `Function` which might take `JobConf` .. @kokosing ?

Reply:
@shandeep00 This change is to ensure that the fileFormat generation logic will be closer to this enum so that in future if we have a new fileFormat it can be added in a single place